packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -171,11 +171,16 @@ export default class PciStoragesContainersAddController {
   }
 
   add() {
+    const containerOffer =
+      this.container.offer === 'storage' ? 'standard' : this.container.offer;
+    const containerType = this.container.containerType
+      ? `${this.container.containerType}::`
+      : '';
```

Comment:
**nitpick:** I see that if `containerType` does not exist, it is set to `standard` in the below code. So combine it.
```suggestion
    const containerType = this.container.containerType
      ? `${this.container.containerType}::`
      : 'standard';
```